src/main/java/org/jabref/logic/importer/fetcher/DoiFetcher.java

```diff
@@ -59,6 +76,40 @@ public Optional<HelpFile> getHelpPage() {
         return Optional.of(HelpFile.FETCHER_DOI);
     }
 
+    private void doAPILimiting(String identifier) {
```

Comment:
I did not consider mEDRA DOI calls because I could not find any information regarding that limiting on their pages (and checking their response headers). I also tried containg them via email about the matter, which they did promptly responded, but only with additional links and information about their API general uses and no sight of an explicit limit.

So, because I concluded the cenario where an user could go over their "non-explicit API limit" was unlikely (needed to be using JabRef´s internal search functionality on ArXiv, which only retrieves a small sample of entries at a time, and that on itself takes some time, and only some of those entries will containt a user-assigned DOI that uses mEDRA), I think leaving behind this limiter is not such a big deal. 

And if a request to mEDRA does not go throw because of that, if they return a 4xx or 5xx error (I´ve seen this behavior from other Agencies APIs, but I didn´t test for mEDRA), the ArXiv Fetcher should just ignore the additional information  from this source and rely on what I could fetch from ArXiv itself and their assigned DOIs (which uses DataCite).